File: app/src/main/java/com/shixun/android/leaving_detection/Detection/Message.java
package com.shixun.android.leaving_detection.Detection;

/**
 * Created by shixunliu on 22/4/17.
 */

public class Message {

    private String message;

    public Message(String str) {
        this.message = str;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: app/src/main/java/com/shixun/android/leaving_detection/DataCollection/Message.kt
package com.shixun.android.leaving_detection.DataCollection

/**
 * Created by shixunliu on 22/4/17.
 * The message sending between fragment or activity by using EventBus
 */

class Message(var message: String?,
              var pressure: Double?,
              var magnetic: Double?,
              var temperature: Double?,
              var wifi: Double?,
              var isWalking: Boolean) {
}
